Periodically-Syncing-HyperChains.md

```diff
@@ -637,15 +637,18 @@ The BFT voting process involves three main phases: Proposal, Voting, and Finaliz
 
    Validators use spend transactions to cast their votes and commit to the final decision. Each transaction includes an encoded payload specifying the vote or commit.
 
+   Thy also vote on adjusting the next epoch length see [Epochs](#epochs).
+
    **Vote Payload Example:**
 
    ```plaintext
-   vote|epoch:42|block_hash:abc123def456ghi789|validator:ak_2cFaGrYvPgsEwMhDPXXrTj2CsW6XrA...|signature:sg_7bf3c4e5d62a8e...
+   vote|epoch:42|block_hash:abc123def456ghi789|+2|validator:ak_2cFaGrYvPgsEwMhDPXXrTj2CsW6XrA...|signature:sg_7bf3c4e5d62a8e...
    ```
 
    - **Type**: `"vote"` indicates the transaction is a vote.
    - **Epoch**: The epoch number being voted on.
    - **Block Hash**: The block hash for which the vote is cast.
+   - **Epoch length adjustment**: +/- N blocks. Increase or decrease the next cycle epoch length.
```

Comment:
is there a valid minimum and maximum for this?

Reply:
Yes. Or, no, but there will be  ;).